clients/lsp-ada.el

```diff
@@ -65,7 +91,9 @@
                                       (lsp--set-configuration
                                        (lsp-configuration-section "ada"))))
                   :server-id 'ada-ls
-                  :synchronize-sections '("ada")))
+                  :synchronize-sections '("ada")
+
```

Comment:
Can you remove the extra newline here? Thanks!

Reply:
Done